both/router/routes.js

```diff
@@ -8,15 +8,26 @@ Router.route('/teach');
 
 Router.route('/learn');
 
+Router.route('/learn/:tag', {
+  subscriptions: function() {
+    return Meteor.subscribe('taggedCourses', this.params.tag)
```

Comment:
Avoid router level subscriptions. Instead, [add subscriptions at the template level](http://guide.meteor.com/data-loading.html#organizing-subscriptions).